client/src/pages/guide/english/network-engineering/osi-layers/index.md

```diff
@@ -31,7 +31,7 @@ In the diagram above, to the extreme left is the unit of data that is used in ea
 
 * _**Layer 1 - Physical Layer:**_  The physical layer is the lowest of the OSI Layers and the most complex. This is because of the undelying hardware technologies used. The function of this layer is to define how the bit stream will be transmitted rather than the logical data packet. It deals with defining which frequency will the bit be transmitted on, what kind of modulation will be used, how the bits will be grouped and other low lying physical parameters needed for transmission of bits.
 
-* _**Layer 2 - Data Link Layer:**_ The data link layer is responsible for transferring data to adjacent devices on the same Local Area Network (LAN). This layer also has provisions to make sure that error-free data is being passed on to the higher layers from the physical layer. Hence, it conteains error detection and correction mechanisms to ensure the integrity of data is maintained.
+* _**Layer 2 - Data Link Layer:**_ The data link layer is responsible for transferring data to adjacent devices on the same Local Area Network (LAN). This layer also has provisions to make sure that error-free data is being passed on to the higher layers from the physical layer. Hence, it conteains error detection and correction mechanisms to ensure the integrity of data is maintained. Data at this layer is commonly referred to as a "frame". The transmission of frames at layer 2 is known as "switching", and is the common function of a Network Switch.
```

Comment:
In addition to the spelling error noted, I noticed you have two other PRs open that make changes to this file (#18380  and #18382). I'm going to close those PRs and ask that you make the changes from them in this PR instead - it'll help prevent merge conflicts and keep us organized. Thanks!